server/src/app.py

```diff
@@ -68,16 +68,17 @@ def configure_app():
     return json(data)
 
 
-@app.route('/requestfrequency')
+@app.route('/requestfrequency', methods=["POST"])
 @compress.compress()
 async def requestfrequency(request):
-    freq_worker = frequency(app.config['Settings'])
+    freq_worker = FrequencyService(app.config['Settings'])
```

Comment:
Right now this is using hardcoded values, let throw in the logic to pull parameters from the request object